openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/flagencoders/bike/CommonBikeFlagEncoder.java

```diff
@@ -245,6 +246,10 @@ protected CommonBikeFlagEncoder(int speedBits, double speedFactor, int maxTurnCo
         avoidHighwayTags.add(KEY_MOTORWAY);
         avoidHighwayTags.add(KEY_MOTORWAY_LINK);
 
+        // bridleways are allowed to ride over in some cases
+        setHighwaySpeed("bridleway", 6);
```

Comment:
As we already introduce `KEY_BRIDLEWAY`, how about using it instead of `"bridleway"?

Reply:
oops, overlooked that

Reply:
No worries! :)